guides/common/attributes-foreman-deb.adoc

```diff
@@ -0,0 +1,31 @@
+// URLs
+:BaseFilenameURL: index-{build}.html
+:AdministeringDocURL: {BaseURL}Administering_Red_Hat_Satellite/{BaseFilenameURL}#
+:ConfiguringAnsibleDocURL: {BaseURL}Configuring_Ansible/{BaseFilenameURL}#
+:ConfiguringLoadBalancerDocURL: {BaseURL}Configuring_Load_Balancer/{BaseFilenameURL}#
+:ContentManagementDocURL: {BaseURL}Content_Management_Guide/{BaseFilenameURL}#
+:InstallingProjectDocURL: {BaseURL}Installing_Server_on_Red_Hat/{BaseFilenameURL}#
+:InstallingSmartProxyDocURL: {BaseURL}Installing_Proxy_on_Red_Hat/{BaseFilenameURL}#
+:ManagingHostsDocURL: {BaseURL}Managing_Hosts/{BaseFilenameURL}#
+:PlanningDocURL: {BaseURL}Planning_Guide/{BaseFilenameURL}#
+:ProvisioningDocURL: {BaseURL}Provisioning_Guide/{BaseFilenameURL}#
+
+// Attributes only for foreman-deb build
+:smart-proxy-installation-guide-title: Installing an External Smart Proxy Server on Debian
+
+// Overrides for foreman-deb build
+:foreman-installer-package: foreman-installer
+:installer-scenario-smartproxy: foreman-installer --no-enable-foreman
+:installer-scenario-smartproxy: foreman-installer --no-enable-foreman
+:installer-scenario: foreman-installer
+:installer-scenario: foreman-installer --scenario foreman
+:package-clean: apt-get clean
+:package-install-project: apt-get install
+:package-install: apt-get install
```

Comment:
This appears in the planning guide unexpectedly? 

`# apt-get install grub2-efi`